reference/docs-conceptual/learn/experimental-features.md

```diff
@@ -73,6 +74,45 @@ In this example, a job is started and a breakpoint is set to break when the `Set
 run. The runspace is stored in a variable and passed to the `Get-PSBreakPoint` command with the
 **Runspace** parameter. You can then inspect the breakpoint in the `$breakpoint` variable.
 
+## PSAnsiRendering
+
+In PowerShell 7.2, this experiment enables the `$PSStyle` automatic variable to control ANSI
+rendering of string output.
+
+```powershell
+PS> $PSStyle
+
+Name            MemberType Definition
+----            ---------- ----------
+AttributesOff   Property   string AttributesOff {get;set;}
+Background      Property   System.Management.Automation.PSStyle+BackgroundColor Background {get;set;}
+Blink           Property   string Blink {get;set;}
+BlinkOff        Property   string BlinkOff {get;set;}
+Bold            Property   string Bold {get;set;}
+BoldOff         Property   string BoldOff {get;set;}
+Foreground      Property   System.Management.Automation.PSStyle+ForegroundColor Foreground {get;set;}
+Formatting      Property   System.Management.Automation.PSStyle+FormattingData Formatting {get;set;}
+Hidden          Property   string Hidden {get;set;}
+HiddenOff       Property   string HiddenOff {get;set;}
+OutputRendering Property   System.Management.Automation.OutputRendering OutputRendering {get;set;}
+Reverse         Property   string Reverse {get;set;}
+ReverseOff      Property   string ReverseOff {get;set;}
+Standout        Property   string Standout {get;set;}
+StandoutOff     Property   string StandoutOff {get;set;}
+Underlined      Property   string Underlined {get;set;}
+UnderlinedOff   Property   string UnderlinedOff {get;set;}
+```
+
+The base members return ANSI escape sequences mapped to their names. These are also settable so you
+can change bold to underlined, for example. This makes it easier for scripters to author decorated
+strings with tab completion:
+
+```powershell
+"$($PSStyle.Background.LightCyan)Power$($PSStyle.Underlined)$($PSStyle.Bold)Shell$($PSStyle.AttributesOff)"
+```
+
+For more information, see [about_automatic_variables](/reference/7.2/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md)
```

Comment:
It's probably worth mentioning that for C# developers, they need to access `PSStyle` which is a singleton, so usage would be like:

```c#
string output = $"{PSStyle.Instance.Foreground.Red}{PSStyle.Instance.Bold}Hello{PSStyle.Instance.Reset}";
```

`PSStyle` exists in the System.Management.Automation namespace.

Reply:
Added